newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -80,6 +97,10 @@ export const GameDetailsDialog = ({
   );
   const [isGameMetricsLoading, setIsGameMetricsLoading] = React.useState(false);
 
+  const [gameBuilds, setGameBuilds] = React.useState<?(Build[])>(null);
+  const [isGameBuildsLoading, setIsGameBuildsLoading] = React.useState(false);
```

Comment:
Could you consider no error (null) and no builds (null, NOT an empty list) as loading? Would avoid one state.